lib/translator/read_capacity_10.cc

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "read_capacity_10.h"
+
+namespace translator {
+
+StatusCode ReadCapacity10ToNvme(absl::Span<const uint8_t> raw_scsi,
+                                nvme::GenericQueueEntryCmd& identify_ns,
+                                scsi::LunAddress lun, Allocation allocation) {
```

Comment:
Do we have documentation on how `Allocation` is used, and how it related to `identify_ns`? It's unclear to me what we use the allocation object for.

Reply:
for quick context, `Allocation` is a way to store and manage all the memory that is allocated so we can properly deallocate it at the end each translation. See this [PR](https://github.com/googleinterns/scsi2nvme/pull/105/files#diff-e8eb0eb4e8ba5e2a5b4c9411b3673f7dR41). I'm working to add more documentation in our `SCSI2NVMe Translation Spec`